cmd/aws-k8s-tester/eks/scale-mng.go

```diff
@@ -0,0 +1,31 @@
+package eks
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+var (
+	name string
+)
+
+func newScaleMNG() *cobra.Command {
```

Comment:
Let's not worry about command right now. We can make it work as built-in add-on (everything runs when cluster creation and mng creation happens)